charts/goldpinger/values.yaml

```diff
@@ -88,6 +88,6 @@ serviceMonitor:
   enabled: false
   selector:
     prometheus: "kube-prometheus"
-  namespace: "monitoring"
+  namespace:
```

Comment:
I'd set
`#  namespace: monitoring`

Reply:
Okay for me. Do you prefer rebasing or just additional commits?

Reply:
Feel free to add commits